cisco_aci/datadog_checks/cisco_aci/models.py

```diff
@@ -0,0 +1,71 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import six
+
+if six.PY3:
+    from typing import Optional
+
+    from pydantic import BaseModel, Field, computed_field
+
+    class NodeAttributes(BaseModel):
+        address: Optional[str] = None
+        ad_st: Optional[str] = Field(default=None, alias="adSt")
+        role: Optional[str] = None
+        dn: Optional[str] = None
+        model: Optional[str] = None
+        version: Optional[str] = None
+        serial: Optional[str] = None
+        vendor: Optional[str] = Field(default='cisco_aci')
+        namespace: Optional[str] = Field(default='default')
+
+    class Node(BaseModel):
+        attributes: NodeAttributes
+
+    class EthAttributes(BaseModel):
+        admin_st: Optional[str] = Field(default=None, alias="adminSt")
+        id: Optional[str] = None
+        name: Optional[str] = None
+        desc: Optional[str] = None
+        router_mac: Optional[str] = Field(default=None, alias="routerMac")
+
+    class Eth(BaseModel):
+        attributes: EthAttributes
+
+    class DeviceMetadata(BaseModel):
+        device_id: Optional[str] = Field(default=None)
+        id_tags: list = Field(default_factory=list)
+        tags: list = Field(default_factory=list)
+        name: Optional[str] = Field(default=None)
+        ip_address: Optional[str] = Field(default=None)
+        model: Optional[str] = Field(default=None)
+        ad_st: Optional[str] = Field(default=None, exclude=True)
```

Comment:
This is just the admin status right? Is there another field that tells us whether or not the device is really up/down?

Reply:
<img width="576" alt="image" src="https://github.com/DataDog/integrations-core/assets/9274242/16a0f089-3de5-4e2b-98b8-93b9b7d0c73f">
https://pubhub.devnetcloud.com/media/apic-mim-ref-301/docs/MO-fabricNode.html#diagram
(reference for the device status)